crates/bevy_pbr/src/pbr_material.rs

```diff
@@ -472,6 +472,9 @@ pub struct StandardMaterial {
     /// Default is [`DEFAULT_PBR_DEFERRED_LIGHTING_PASS_ID`] for default
     /// PBR deferred lighting pass. Ignored in the case of forward materials.
     pub deferred_lighting_pass_id: u8,
+
+    /// Texture UV transform.
+    pub uv_transform: Mat3,
```

Comment:
Agree as @mockersf previously suggested that this would be better as an `Affine2`. The cost of converting to `Mat3` when converting to `StandardMaterialUniform` is [basically free](https://docs.rs/glam/latest/src/glam/f32/affine2.rs.html#396-405).

Reply:
[`7298cdb` (#11904)](https://github.com/bevyengine/bevy/pull/11904/commits/7298cdbe62891229c8e99edff358a51757ccc941)